cli/module_registry.go

```diff
@@ -699,6 +699,11 @@ func readModels(path string, logger logging.Logger) ([]ModuleComponent, error) {
 	defer vutils.UncheckedErrorFunc(func() error { return os.RemoveAll(parentAddr) })
 	parentAddr += "/parent.sock"
 
+	// in some scenarios, we want to run module code without having a "real"
+	// parent available. This environment variable illustrates this.
+	os.Setenv("VIAM_NO_MODULE_PARENT", "true")
```

Comment:
linter was flagging this - `os.Setenv` can fail so we need to check this

```suggestion
	if err := os.Setenv("VIAM_NO_MODULE_PARENT", "true"); err != nil {
		return nil ,err
	}
```